spec/vcr_configurer.rb

```diff
@@ -32,7 +32,9 @@ def self.filter_header(vcr_config, part, header_name)
         vcr_config.configure_rspec_metadata!
         vcr_config.default_cassette_options = {
           :decode_compressed_response => true,
-          :match_requests_on => [:path, :method, :query]
+          :match_requests_on => [:path, :method, :query],
+          # allow to set record mode from environment, see https://relishapp.com/vcr/vcr/v/3-0-3/docs/record-modes
+          :record => (ENV['VCR_RECORD_MODE'] && ENV['VCR_RECORD_MODE'].to_sym) || :once
```

Comment:
This allows us to set the record mode. However, we cannot fully disable recording. Would it be useful to add this option?